common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -194,26 +198,23 @@ def test_import_timestamp(self):
             And if I refresh the page, the timestamp is still displayed
         """
         self.assertFalse(self.import_page.is_timestamp_visible())
+
+        # Get the time when the import has started.
+        upload_start_time = datetime.utcnow().replace(microsecond=0, second=0)
         self.import_page.upload_tarball(self.tarball_name)
         self.import_page.wait_for_upload()
 
-        utc_now = datetime.utcnow()
-        import_date, import_time = self.import_page.timestamp
+        # Get the time when the import has finished.
+        upload_finish_time = datetime.utcnow().replace(microsecond=0, second=0)
 
+        import_timestamp = self.import_page.parsed_timestamp
         self.import_page.wait_for_timestamp_visible()
-        # Flaky pattern:
-        #    This test failed because the utc_now and import date
-        #    might be assigned at different times. The error message
-        #    was "'18:30' != u'18:29'", meaning it uploaded it at 18:29, then,
-        #    when we assigned utc_now, the time had crossed the minute to
-        #    18:30.
-        # Possible fixes:
-        # *  Mock utcnow somehow.
-        # *  Check for the date and time within a certain range, rather than
-        #    doing a string comparison.
-        self.assertEqual(utc_now.strftime('%m/%d/%Y'), import_date)
-        self.assertEqual(utc_now.strftime('%H:%M'), import_time)
 
+        # Verify that 'import_timestamp' is between start and finish upload time
+        self.assertTrue(upload_start_time <= import_timestamp,
```

Comment:
you can use https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertLessEqual

Also, use the python style guidelines: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines